CHANGES.md

```diff
@@ -18,6 +18,7 @@ Apollo 2.2.0
 * [[Multi-Database Support] Optimize column define case sensitivity](https://github.com/apolloconfig/apollo/pull/4776)
 * [[Multi-Database Support][pg] Where clause need escape, otherwise will request postgre use lowwer case](https://github.com/apolloconfig/apollo/pull/4780)
 * [Misc dependency updates](https://github.com/apolloconfig/apollo/pull/4784)
+* [[HealthIndicator] Refactor health indicator]
```

Comment:
Change the description?
```suggestion
* [refactor: AdminServiceHealthIndicator extends AbstractHealthIndicator instead of implements HealthIndicator](https://github.com/apolloconfig/apollo/pull/4801)
```
and the title